deploy/roles/postgresql-server-config/tasks/main.yml

```diff
@@ -3,67 +3,110 @@
   yum:
     name: "{{ item }}"
     state: present
+  become: yes
+  become_method: sudo
   with_items:
-    - postgresql94-devel
+    - postgresql96-devel
     - python-psycopg2
     - git
-  always_run: true
+  check_mode: no
+
+- name: Allow postgres empty password login temporarily
+  become: yes
+  become_method: sudo
+  lineinfile:
+    dest: "{{ pp_datadir }}/pg_hba.conf"
+    regexp: '^(.*)::1/128(.*)md5'
+    backrefs: yes
+    line: '\1::1/128\2trust'
+  notify: restart postgres
+  when: install_postgres
+
+- meta: flush_handlers
+
+- name: Ensure password is correct
+  postgresql_exec:
+    login_host: localhost
+    login_user: "{{ pp_superaccount }}"
+    port: "{{ pp_serverport }}"
+    db: postgres
+    script: "ALTER user postgres PASSWORD '{{ pp_superpassword }}';"
+  check_mode: no
+  when: install_postgres
+
+- name: Disallow postgres empty password login
+  become: yes
+  become_method: sudo
+  lineinfile:
+    dest: "{{ pp_datadir }}/pg_hba.conf"
+    regexp: '^(.*)::1/128(.*)trust'
+    backrefs: yes
+    line: '\1::1/128\2md5'
+  when: install_postgres
+  notify: restart postgres
+
+- meta: flush_handlers
```

Comment:
Before, I had changed the pg_hba.conf entry to 'trust' to enable postgres to be able to login since postgres user doesn't have a password initially. This way I can change it back to md5 and the rest of the playbook will still work, since we added a password.

Reply:
Ah, got it, thanks!